terraform/modules/jenkins/s3_bucket.tf

```diff
@@ -0,0 +1,28 @@
+resource "aws_s3_bucket" "jenkins_logs_bucket" {
+  bucket        = "${var.name}-${var.dns_name}-logs-bucket"
+  force_destroy = true
+
+  policy = <<POLICY
+{
+    "Version": "2012-10-17",
```

Comment:
We have `Version: 2012-10-17` all over our terraform stuff and I am not sure what it refers to. Terraform version? The policy version? AWS resource version?

Reply:
I know right!

I think it's policy document specification version. It's in all the AWS documentation examples. We could try leaving it out...

Reply:
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_version.html

Reply:
Reading The Fine Documentation reveals that if we don't specify a version, we lose out on newer features... 🤷‍♀️